tests/unit/test_module_names.py

```diff
@@ -91,8 +91,14 @@ def test_module_name(self):
                 path, directory, filename.split("_test")[0]
             )
 
-        error_msg += "\nIf you believe one of the entries above should be ignored, please add it to either\n"
-        error_msg += "'EXCLUDED_DIRS' or 'EXCLUDED_FILES' in 'tests/unit/test_module_names.py'.\n"
+        error_msg += (
```

Comment:
Not a blocker for this but could we just combine all these lines into one addition?

Reply:
Will be fixed on next push